pkg/apis/kubermatic/v1/preset.go

```diff
@@ -326,11 +326,11 @@ type Openstack struct {
 	ApplicationCredentialID     string `json:"applicationCredentialID,omitempty"`
 	ApplicationCredentialSecret string `json:"applicationCredentialSecret,omitempty"`
 
-	Username string `json:"username"`
-	Password string `json:"password"`
-	Tenant   string `json:"tenant"`
-	TenantID string `json:"tenantID"`
-	Domain   string `json:"domain"`
+	Username  string `json:"username"`
+	Password  string `json:"password"`
+	Project   string `json:"project"`
+	ProjectID string `json:"projectID"`
```

Comment:
Remember to adjust presets so dev and CI will not break.

Reply:
@maciaszczykm  beside `hack/ci/run-api-e2e.sh` there is any other adjustment to do?

Reply:
The presets that are defined in our Vault should be updated to use the new field once this merges. It should not break anything until there is fallback though.